src/v/pandaproxy/json/requests/fetch.h

```diff
@@ -102,6 +108,8 @@ class rjson_serialize_impl<kafka::fetch_response> {
             }
         }
         w.EndArray();
+
+        return true;
```

Comment:
Where does false get returned? Should the return on line 106 be checked?

Reply:
Added check on line 106.  No where else does it return false, but I had to update all of the signatures.

Reply:
Maybe I'm blind, but I don't see that check anywhere.

Reply:
Sorry line 123